src/applications/facility-locator/utils/facilityDistance.js

```diff
@@ -60,3 +61,25 @@ export function calculateBoundingBox(lat, long, radius) {
     toDegrees(maxLatitude).toFixed(3),
   ];
 }
+
+export const convertMetersToMiles = meters =>
+  Math.round(meters * 0.000621371192);
+
+export const distancesToNearbyVetCenters = (
+  originalVetCenterCoordinates,
+  nearbyVetCentersCoordinates,
+) => {
+  const originalVetCenterCoordinatesParam = originalVetCenterCoordinates.join(
+    ',',
+  );
+  const startingCoordinatesParam = nearbyVetCentersCoordinates
+    .map(loc => loc.join(','))
+    .join(';');
+  let sourcesParam = '';
+
+  for (let i = 1; i <= nearbyVetCentersCoordinates.length; i += 1) {
```

Comment:
Looks like you might be able to just use `all` as the value for sources. Then you wouldn't need this for loop. https://docs.mapbox.com/api/navigation/matrix/

Reply:
I tried that before but it was polluting the data. It included the coordinates for the original vet center, so a distance of `0` was getting mixed into the returned data.